### PR TITLE
SOW-24 add link to visualisation example

### PR DESCRIPTION
**Pull request by vferraro-scottlogic**
## Changes
- add link to visualisation in navbar 
- add header and paragraph to visualisation example page

### CODE REVIEW

**Comment by vferraro-scottlogic** [comment]
> We also need to do the last AC for updating the route

I had forgotten to update that. Thanks Ronan :). Done now

File: src/pages/visualisation-example/VisualisationExamplePage.tsx [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
 export const VisualisationExamplePage = (): JSX.Element => {
   return (
-    <Visualisation
-      developmentPlanFilepath={"https://raw.githubusercontent.com/digital-land/local-plans-timetable/main/assets/development-plan.csv"}
-      timetableEventsFilepath={"https://raw.githubusercontent.com/digital-land/local-plans-timetable/main/assets/timetable.csv"}
-    />
+    <NavPageWrapper pageTitle="What a Local Plan timetable created online looks like">
+      <p className="govuk-body">
+        This is an{" "}
+        <strong className={`govuk-!-font-weight-bold`}>example</strong> of what
```
Comment (Ro4052) [inline src/pages/visualisation-example/VisualisationExamplePage.tsx:9]: Out of interest, do we need this class?

I'm also going to create a ticket for changing the existing `b` tags to `strong` (`strong` is better for accessibility)

**Review by Ro4052 — DISMISSED** [review]

File: src/pages/visualisation-example/VisualisationExamplePage.tsx (hunk 1 above)
Comment (vferraro-scottlogic) [reply src/pages/visualisation-example/VisualisationExamplePage.tsx]: actually we don't need it. I have removed it 👍

Comment (vferraro-scottlogic) [reply src/pages/visualisation-example/VisualisationExamplePage.tsx]: I ve used it for the Tag component as <strong/> there wasnt displaying bold text but here seems to work fine. (I think for Tag there was a conflict with the "govuk-tag" className )

**Review by Ro4052 — APPROVED** [review]

**Pull request merged**